### DISCUSSION TITLE
How to set the Zigbee channel with Zigbee Tasmota and Home Assistant?

### DISCUSSION THREAD

**Discussion by kiat-huang** [arendst/Tasmota#11013] [Q&A] [answered]
I've a Sonoff Zigbee Bridge flashed with the latest build "Version":"9.2.0.7(zbbridge)".  I'm running a second DeConz Zigbee controller on one channel and I'd like to see and if need be set the current channel on the TasmotaZB.  How to do that when linked to Home Assistant?

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you. Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
